File: URI_1044_multiples.js
const lines = ["6 25"];

const inputArr = lines[0].split(" ");
const a = Number(inputArr[0]);
const b = Number(inputArr[1]);

if (a%b===0 || b%a===0) {
    console.log(`Sao Multiplos`);
} else {
    console.log(`Nao sao Multiplos`);
}

File: URI_1044_multiples.java
import java.util.Scanner;

public class URI_1044_multiples {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		float a = scanner.nextFloat();
		float b = scanner.nextFloat();

		if (a % b == 0 || b % a == 0) {
			System.out.println("Sao Multiplos");
		} else {
			System.out.println("Nao sao Multiplos");
		}

		scanner.close();
	}
}